.github/workflows/publish.yml

```diff
@@ -24,14 +24,21 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Publish Docker images
         run: make publish-containers
-
+      - name: Publish Draconctl image
+        run: make draconctl-image-publish
+      - name: Setup Go
+        uses: actions/setup-go@v4
       - name: Publish component Helm package
         run: |
-          set -e;
-          DRACON_VERSION_SEMVER=$(sed 's/v//' <<< ${{ github.ref_name }});
-          make cmd/draconctl/bin; \
-          bin/cmd/draconctl components package --version ${{ github.ref_name }} --chart-version ${DRACON_VERSION_SEMVER} --name dracon-oss-components ./components; \
+          set -e
+          DRACON_VERSION_SEMVER=$(sed 's/v//' <<< ${{ github.ref_name }})
+
+          go install github.com/ocurity/dracon/cmd/draconctl@latest
```

Comment:
why do this? you already checked out the latest version of the codebase and you are executing commands in it

Reply:
it's a matter of principle:
in this scope draconctl is a tool used to package and publish things,  hence we use the latest stable version

if we were to be testing draconctl then we can build it from the binary
i don't see how this is a problem

Reply:
you are not using the latest stable version, you are using the latest commit in the main branch. these are not the same unfortunately